### ISSUE TITLE
Using tabs in scilla code make error

### ISSUE THREAD

**Issue by onokatio** [Zilliqa/kaya#44]
# Problem

When use tab and Kaya and zilliqa-js, deploy will fail.
Interpreter says:

```
{ id: 1,
  jsonrpc: '2.0',
  result:
   { Error:
      'Interpreter error: Syntax error in file tmp/967c182975d2046a7f5bcbae87b625761e7757a3_code.scilla: line 1, position 71.: Unexpected character: \\\n' } }
```
 Content of `tmp/967c182975d2046a7f5bcbae87b625761e7757a3_code.scilla` is here:

```
import BoolUtils  library Room  let zero = Uint128 0  let one_msg =  \tfun ( ...
```

position 71 is '\t'.

# How to resolve 

I replace tab to 4 spaces.

**Comment by onokatio** [comment]
my guess reson, Brake line (\n) will be replace nothing , but \n\t will not replaced.

**Comment by iantanwx** [comment]
Fixed by #45
